CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [[Unreleased]]
 ### Added:
 - Function to parse the final account balances from a transaction's metadata
+- Support for ED25519 seeds that don't use the `sEd` prefix
```

Comment:
```suggestion
- Support for Ed25519 seeds that don't use the `sEd` prefix
```
I usually see the `d` in lowercase - presumably because it is an abbreviation for [Edwards](https://en.wikipedia.org/wiki/Edwards_curve)